app/src/androidTest/java/com/example/myapplication/ServerInfoApiInstrumentedTest.kt

```diff
@@ -0,0 +1,57 @@
+package com.example.myapplication
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import edu.usf.cutr.otp.serverinfo.api.ServerInfoApi
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers.Main
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.launch
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import kotlin.coroutines.CoroutineContext
+
+@RunWith(AndroidJUnit4::class)
```

Comment:
These instrumented Android tests are nice, but they are also hard to run on CI because running an emulator on CI is hard.

Let's pause this PR and finish the JVM build (desktop Java) of this project first (https://github.com/CUTR-at-USF/opentripplanner-client-library/issues/6), and then we can come back and run these unit tests from the JVM build, which will make CI much easier to manage and much faster to complete.

Reply:
After more research, it seems this may help solve the issue of running an emulator on GitHub Actions and it's used by Square and others:
https://github.com/marketplace/actions/android-emulator-runner

@BumbleFlash If you keep hitting issues getting the coroutines to work right in JVM tests feel free to try this with Android instrumented tests